torch/nn/functional.py

```diff
@@ -2113,9 +2113,20 @@ def embedding(
 
     if has_torch_function_variadic(input, weight):
         return handle_torch_function(
-            embedding, (input, weight),
-            input, weight, padding_idx, max_norm, norm_type,
-            scale_grad_by_freq, sparse
+            embedding,
+            (input, weight),
+            input,
+            weight,
+            padding_idx,
+            max_norm,
+            norm_type,
+            scale_grad_by_freq,
+            sparse,
+            padding_idx=padding_idx,
+            max_norm=max_norm,
+            norm_type=norm_type,
+            scale_grad_by_freq=scale_grad_by_freq,
+            sparse=sparse,
```

Comment:
I don't think passing both positional and kwarg forms is what we want - kwarg-only should be okay.

Unfortunately, whether args are passed positionally or as kwargs doesn't seem to be very consistent across functions in `torch.nn.functional` atm but ideally it would be.

Reply:
Do you feel that we should use kwarg for all ops? So basically in the shareded embedding op. If we don't pass in via kwarg. All the args will not be sent into `__torch_function__ `. Sharded embedding was added in this PR:  https://github.com/pytorch/pytorch/pull/66604. I am working on a PR which enables more input params for both Sharded Embedding and Embedding Bag. However, the parameters are only passed in for the embeddingBag case.

Reply:
Well, I can get these from positional args but this kindly make the code look hacky and bringing in implicit dependency. So I think we might want to pass in via kargs and use the key to get the parameter like max_norm, padding_idx, etc.